ext/pdo_odbc/config.m4

```diff
@@ -1,26 +1,29 @@
 define([PDO_ODBC_HELP_TEXT],[[
-  The include and lib dirs are looked for under 'dir'. The 'flavour' can be one
-  of: ibm-db2, iODBC, unixODBC, generic. If ',dir' part is omitted, default for
-  the flavour you have selected will be used. e.g.: --with-pdo-odbc=unixODBC
-  will check for unixODBC under /usr/local. You may attempt to use an otherwise
-  unsupported driver using the 'generic' flavour. The syntax for generic ODBC
-  support is: --with-pdo-odbc=generic,dir,libname,ldflags,cflags. When built as
-  'shared' the extension filename is always pdo_odbc.so]])
+  The 'flavour' part determines what driver or driver manager to use; it can be
+  either ibm-db2, iODBC, unixODBC, or generic.
+
+  The include and lib dirs are looked for under 'dir'.  You may attempt to use
+  an otherwise unsupported driver using the 'generic' flavour. The syntax for
+  generic ODBC support is: --with-pdo-odbc=generic,dir,libname,ldflags,cflags.
+  When built as 'shared' the extension filename is always 'pdo_odbc.so'.
+
+  For unixODBC and iODBC, the 'dir' part is ignored and can be omitted, as
+  pkg-config will be searched instead. For ibm-db2, it will search for the DB2
+  driver at that path.]])
 
 PHP_ARG_WITH([pdo-odbc],
   [for ODBC v3 support for PDO],
   [AS_HELP_STRING([--with-pdo-odbc=flavour,dir],
     [PDO: Support for 'flavour' ODBC driver.]PDO_ODBC_HELP_TEXT)])
 
 AC_DEFUN([PHP_PDO_ODBC_CHECK_HEADER],
-[AC_MSG_CHECKING([for $1 in $PDO_ODBC_INCDIR])
-  AS_IF([test -f "$PDO_ODBC_INCDIR/$1"], [
+[AC_MSG_CHECKING([for $1])
+  AC_PREPROC_IFELSE([AC_LANG_PROGRAM([#include <$1>], [])], [
     php_pdo_odbc_have_header=yes
     AC_DEFINE_UNQUOTED(AS_TR_CPP([HAVE_$1]), [1],
       [Define to 1 if you have the <$1> header file.])
-    AC_MSG_RESULT([yes])
-  ],
-  [AC_MSG_RESULT([no])])
+      AC_MSG_RESULT([yes])
+  ], [AC_MSG_RESULT([no])])
```

Comment:
```suggestion
    AC_MSG_RESULT([yes])
  ],
  [AC_MSG_RESULT([no])])
```

That was just some unrelated change in that diff in my comment. Perhaps to slightly keep the CS.